buildbot/master/files/config/environments.py

```diff
@@ -31,6 +31,7 @@ def __add__(self, other):
 
 build_windows = build_common + Environment({
     'CARGO_HOME': '/home/Administrator/.cargo',
+    'HOME': r'C:\buildbot\slave\windows\build',
```

Comment:
That's a reasonable hack, but please leave a comment with that explanation for future reference.